src/heltour.js

```diff
@@ -1,16 +1,20 @@
 //------------------------------------------------------------------------------
 // Heltour related facilities
 //------------------------------------------------------------------------------
-
+const _ = require('lodash');
 const url = require('url');
 const http = require("./http.js");
 
-function findPairing(heltourConfig, white, black) {
+function findPairing(heltourConfig, white, black, league_tag) {
```

Comment:
ill add a comment. the intent was to allow calling findPairing without it - as you noted, this cannot be done by relying on the config.